launchable/commands/record/commit.py

```diff
@@ -1,7 +1,16 @@
-from ...utils.token import parse_token
 import os
 import click
 from ...utils.ingester_image import ingester_image
+from ...utils.ingester_jar import ingester_jar_url, ingester_jar_version
+import subprocess
+import requests
+import pathlib
+import sys
+
+
+jar_dir_path = os.path.expanduser(
+    "~/.launchable/jar/ingester/{}".format(ingester_jar_version))
```

Comment:
I specified `~/.launchable/jar/ingester/0.1.0/exe_deploy.jar` as a local jar file path. Is this okay?